roverrobotics_driver/launch/pro.launch

```diff
@@ -3,16 +3,16 @@
     <arg name="config_locks" default="$(find roverrobotics_driver)/config/twist_mux_locks.yaml" />
     <arg name="config_topics" default="$(find roverrobotics_driver)/config/twist_mux_topics_locks.yaml" />
     <param name="device_port" value="/dev/rover" />
-    <param name="motor1_id" value="1" />
-    <param name="motor2_id" value="2" />
-    <param name="motor3_id" value="3" />
-    <param name="motor4_id" value="4" />
     <param name="comm_type" value="serial" />
     <param name="robot_type" value="pro" />
     <param name="status_frequency" value="30" />
     <param name="closed_loop_control" value="false" />
     <param name="Kp" value="0.4" />
     <param name="Ki" value="0.7" />
+    <param name="angular_a" value="0" />
```

Comment:
these values look suspicious. Is angular scaling even implemented on the rover pro? IF SO, then the angular_c value should be 1 by default, and a/b should both be 0.

Reply:
Not sure; would have to check librover;